src/Screens/Login Screen/Login.jsx

```diff
@@ -1,19 +1,65 @@
+import { useState } from "react";
+import { Link, useNavigate, useLocation } from "react-router-dom";
+import { useAuth } from "../../Helper";
 import "./login.css";
-import { Link } from "react-router-dom";
 
 const Login = () => {
+  const { setIsLoggedIn } = useAuth();
+  const navigate = useNavigate();
+  const location = useLocation();
+  const toPreviousRoute = location?.state?.from?.pathname || "/";
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState("");
+  const [showPassword, setShowPassword] = useState(false);
+  const [error, setError] = useState(false);
+
+  const emailHandler = (e) => {
+    e.preventDefault();
+    setEmail(e.target.value);
+    setError(false);
+  };
+
+  const passwordHandler = (e) => {
+    e.preventDefault();
+    setPassword(e.target.value);
+    setError(false);
+  };
+
+  const showPasswordHandler = (e) => {
+    e.preventDefault();
+    setShowPassword((prev) => !prev);
+  };
+
+  const submitHandler = (e) => {
+    e.preventDefault();
+    if (password === "adarshbalika" && email === "adarshbalika@gmail.com") {
```

Comment:
I think the api call will check the test credentials internally , you don't have to do this explicitly.

Reply:
yeah, this needs to be fixed before merging.

Reply:
fixed in https://github.com/saadahmadhere/elmart/pull/13/commits/e14d398edddba37855b47127a491c1dc54370132